services/common/src/components/comments/CommentEditor.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { Input, Button } from "antd";
+import { getUserAccessData } from "@mds/common/redux/selectors/authenticationSelectors";
```

Comment:
I think you can remove a couple imports here- the getUserAccessData and the connect (mapStateToProps isn't being used, can be removed)